test/framework/debug.go

```diff
@@ -272,8 +283,7 @@ const (
 	zoneingressType dpType = "zoneingress"
 )
 
-func configDump(kumactlOpts kumactl.KumactlOptions, debugPath string, cluster Cluster, mesh string, dpType dpType) {
-	errorSeen := false
+func inspectDataplane(kumactlOpts *kumactl.KumactlOptions, debugPath string, cluster Cluster, mesh string, dpType dpType) {
```

Comment:
question. Could we have the same implementation for inspecting for both universal and kube? It should be the same in the end, right?